src/test/ui/type-alias-impl-trait/generic_duplicate_param_use2.stderr

```diff
@@ -7,5 +7,23 @@ LL | |     t
 LL | | }
    | |_^
 
-error: aborting due to previous error
+error: concrete type differs from previous defining opaque type use
```

Comment:
This *does* seem like more confusing output than the old code...

Reply:
The difference is when the checks run and which "uses" (mentions, really) are considered "defining".
From the looks of the code, "defining" vs not is an unimplemented part of the feature.
At least now `check::wfcheck` and `type_of` are each more uniform and it should be easier to finish the feature, but I'd rather leave that to someone else.